recipes/zipline-reloaded/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "zipline-reloaded" %}
+{% set version = "2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zipline-reloaded-{{ version }}.tar.gz
+  sha256: 8c533c1ccf5069715bf2e9bd5e87ea39a3f7c83f5ef2c00d74d7311c8f31ecf8
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - numpy
+    - cython
+    - setuptools-scm
+    - pip
+  run:
+    - python
+    - alembic >=0.7.7
+    - bcolz-zipline >=1.2.6
+    - bottleneck >=1.0.0
+    - click >=4.0.0
+    - empyrical-reloaded >=0.5.7
+    - h5py >=2.7.1
+    - intervaltree >=2.1.0
+    - iso3166 >=2.1.1
+    - iso4217 >=1.6.20180829
+    - lru-dict >=1.1.4
+    - multipledispatch >=0.6.0
+    - networkx >=2.0
+    - numexpr >=2.6.1
+    - pandas <1.6,>=1.3
+    - patsy >=0.4.0
+    - python-dateutil >=2.4.2
+    - python-interface >=1.5.3
+    - pytz >=2018.5
+    - requests >=2.9.1
+    - scipy >=0.17.1
+    - six >=1.10.0
+    - sqlalchemy <2,>=1.0.8a
+    - statsmodels >=0.6.1
+    - ta-lib >=0.4.09
+    - pytables >=3.4.3
+    - toolz >=0.8.2
+    - exchange-calendars >=4.2.4
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - zipline
+  source_files:
+    - tests
+  commands:
+    - pytest --reruns 5 -k "not nothing and not correlation_factors and not simple_beta_target and not test_regression_of_returns_factor"  tests
+  requires:
+    - pytest>=7.2.0
```

Comment:
```suggestion
    - pytest >=7.2.0
```

Reply:
Thanks for your quick reply @marcelotrevisani! 

I had originally submitted these various recipes together (#21700) but was then asked to split them up (on Gitter); the PRs for the three dependencies have since been merged (#21704, #21705, #21706). Should I still add those recipes back here?